develop.md

```diff
@@ -1,49 +1,195 @@
-3. Update the version in `.travis.yml`
-4. Run tests, for example: `./run-tests.sh 7.9-community`
-5. Create a Pull Request, confirm Travis build passes, get it reviewed and merged
+Create a new subdirectory with the major version number in the root directory of the repository with a sub directory for each supported edition, eg.: 
 
-Update the public repositories for [Docker Hub][hub]:
+```
+mkdir -p 9/community 9/developer 9/enterprise
+```
 
-1. Only if necessary, update files in https://github.com/docker-library/docs/tree/master/sonarqube and create a Pull Request
-    1. The markdown format must follow a certain standard, otherwise automated tests will fail. You can test with the `markdownfmt.sh` tool included in the repository, for example `./markdownfmt.sh -d sonarqube/content.md` will output the diff that would have to be done to make the tests pass. You can use the `patch` command to apply the changes, for example: `./markdownfmt.sh -d sonarqube/content.md | patch sonarqube/content.md`
-    2. Verify the Pull Request passes the automated tests (visible in the status of the PR)
+Each edition directory will contain a single DockerFile.
 
-2. Update https://github.com/docker-library/official-images/blob/master/library/sonarqube
-    1. Update the `Tags` and `Directory` entries appropriately
-    2. Update the commit sha to the just merged commit in `docker-sonarqube`
-    3. Create a Pull Request, with a simple summary of the new release
-    4. Verify the Pull Request passes the automated tests (visible in the status of the PR)
+New non-major version
+---------------------
 
-3. Wait for the Pull Requests to get merged, and verify the updated page on [Docker Hub][hub]
+As of today, we publish only a single version of SQ for a given major version in the "master" branch.
 
-FAQ
----
+New new image is therefor created for non-major versions of SQ. Instead, existing images are updated.
 
-Q: Why don't you provide Alpine-based images?
+Docker images of older intermediate versions are accessible via tags.
 
-A: As of today, OpenJDK 8 does not officially support Alpine. As such, we don't know a good base image to build on. We did try the unsupported `openjdk:8-alpine` image as base, and we saw seen nonsense problems with nonsense workarounds. We've left the recipes for Alpine-based images in the branch named `alpine`, build and use at your own risk.
 
-Other notes
------------
+Dogfooding and images for local artifacts
+=========================================
+
+Dogfooding
+----------
+
+SonarQube Docker images are dogfooded.
+
+This implies that Docker images are produced for every artifact produced from the dogfood branch of SonarQube (see the [dogfood_docker_build_task](https://github.com/SonarSource/sonar-enterprise/blob/master/.cirrus.yml#L263)).
+
+Docker images of SonarQube are built from publicly available artifacts, which means the DockerFile can simply download them from the public place they are hosted (`binaries.sonarsource.com`).
```

Comment:
It's always named just`Dockerfile`, no need to uppercase the `file` part (needs changing in a few places).